benchmarks/rigid_shear/time-dependent/creates_formatted_tables.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+grid_resolution=('3' '4' '5' '6')
+interpolators=('cell_average' 'bilinear_least_squares')
+integrators=('rk2' 'rk4')
+stokes_degrees=('2' '3')
+
+for stokes_degree in ${stokes_degrees[@]}; do
+  for interpolator in ${interpolators[@]}; do
+  for integrator in ${integrators[@]}; do
```

Comment:
If you indent line 9 by one character less, then things still look nicely staggered :-) If you don't want to do that, then just indent this and the following lines by two more spaces.